challenge.js

```diff
@@ -27,6 +28,8 @@
 
 // ANSWER
 
+  E is a descendent of A
```

Comment:
This is correct, but very general. This question was testing to see that you'd put two and two together (C is a child of A, D is a sibling of A), so this is a satisfactory answer. To be specific, you could say E is a grandchild of A. 3/4